buildSrc/src/main/kotlin/scripts/infrastructure.gradle.kts

```diff
@@ -13,31 +19,37 @@ tasks.named<Wrapper>("wrapper") {
 
 tasks.register("runUnitTests") {
     description = "Runs all Unit Tests."
-    dependsOn(":app:testDebugUnitTest")
+    dependsOn(":app:test${Default.BUILD_FLAVOR}${Default.BUILD_TYPE}UnitTest")
```

Comment:
`${Default.BUILD_FLAVOR}${Default.BUILD_TYPE}` is repeated too much. maybe we can combine them into 
`val variant = "${BUILD_FLAVOR.capitalize()}${BUILD_TYPE.capitalize()}"`
  in `Default`? 

In that way, we also wouldn't have to make `.toLowerCase()` for `applicationPackage`

Reply:
I do not fully understand your point. 

For ```applicationPackage``` we need to concatenate ```.dev.debug```
For ```gradle``` tasks we need "DevDebug". 

Maybe I'm missing something but I do not see a solution that applies to all with only one constant.

Reply:
```
private object Default {
    val BUILD_TYPE = BuildTypes.DEBUG         <-- removed ".capitalize()"
    val BUILD_FLAVOR = ProductFlavors.DEV         <-- removed ".capitalize()"
    
    val VARIANT = "${BUILD_FLAVOR.capitalize()}${BUILD_TYPE.capitalize()}"     <-- combined the logic here
}


tasks.register("runAcceptanceTests") {
    ...
    dependsOn(":app:connected${Default.VARIANT}AndroidTest")
}


tasks.register("compileApp") {
     ...
    dependsOn(":app:assemble${Default.VARIANT}")
}


tasks.register("runApp", Exec::class) {
    ...
    val applicationPackage = "com.wire.android.${Default.BUILD_FLAVOR}.${Default.BUILD_TYPE}"
}
```

Reply:
Got it! Thanks for clearing it up. I will modify this and merge oki?